plugins/warn_unused/warn_unused_main.ml

```diff
@@ -118,31 +118,33 @@ module Cmdline = struct
     end
   ]
 
-  let info = Term.info ~version ~man ~doc name
   let passes = [name; "--taint"; "--mark"; "--print"]
   let not_a_pass s = not (List.mem passes s)
 
   let pass name =
     let doc = sprintf "run $mname-%s pass" name in
-    Arg.(value & flag & info [name] ~doc)
-  let taint = pass "taint"
-  let print = pass "print"
-  let mark  = pass "mark"
+    Config.(flag name ~doc)
```

Comment:
Do not rename passes, and do not resolve them. Just substitute `ignore_args` with `ignore` and it will work out of box.